test/spec/index.js

```diff
@@ -58,10 +58,15 @@ describe('Clay', function() {
     function() {
       stubPebble();
       var clay = fixture.clay([]);
+
+      // we stub the generateUrl method to avoid very large string comparisons.
+      var generateUrlStub = sinon.stub(clay, 'generateUrl');
+      generateUrlStub.returns('data:text/html;base64,PGh0bWw%2BVEVTVDwvaHRtbD4%3D');
       Pebble.addEventListener.withArgs('showConfiguration').callArg(1);
 
       assert(Pebble.addEventListener.calledWith('showConfiguration'));
       assert(Pebble.openURL.calledWith(clay.generateUrl()));
+      generateUrlStub.restore();
```

Comment:
In general it is not good to do any mocking/stubbing inside of your test conditions because if there is an error or assert failure, the mock/stub won't be restored and subsequent tests will fail as well. Also keep in mind that it could lead to inconsistent state between tests.

I've always manage the mock/stub creation and restoring in beforeEach and afterEach.